2025-05-21 19:45:18,611	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-05-21 19:45:18,930	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 2700
2025-05-21 19:45:19,141	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/users/@me, 错误代码: 401, 返回内容: {'message': '接口访问源IP不在白名单', 'code': 11298, 'err_code': 40023002, 'trace_id': '7289379f2236c5081e6094d4f7a40ce2'}, trace_id:7289379f2236c5081e6094d4f7a40ce2
2025-05-21 19:45:26,971	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-05-21 19:45:27,165	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 2691
2025-05-21 19:45:27,347	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/users/@me, 错误代码: 401, 返回内容: {'message': '接口访问源IP不在白名单', 'code': 11298, 'err_code': 40023002, 'trace_id': '09fe3a0bced03233cc8948a3c016b198'}, trace_id:09fe3a0bced03233cc8948a3c016b198
2025-05-21 19:49:07,562	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-05-21 19:49:07,812	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 2471
2025-05-21 19:49:08,304	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-05-21 19:49:08,304	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-05-21 19:49:08,304	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-05-21 19:49:08,304	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-05-21 19:49:08,410	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-05-21 19:49:08,510	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「The_PRTS-测试中」启动成功！
2025-05-21 19:49:08,511	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-05-21 19:50:09,132	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-05-21 19:50:09,367	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 2409
2025-05-21 19:50:09,822	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-05-21 19:50:09,824	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-05-21 19:50:09,824	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-05-21 19:50:09,824	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-05-21 19:50:09,945	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-05-21 19:50:10,022	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「The_PRTS-测试中」启动成功！
2025-05-21 19:50:10,022	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-05-21 19:53:18,964	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-05-21 19:53:19,212	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 2219
2025-05-21 19:53:19,721	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-05-21 19:53:19,721	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-05-21 19:53:19,721	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-05-21 19:53:19,721	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-05-21 19:53:19,874	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-05-21 19:53:19,971	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「The_PRTS-测试中」启动成功！
2025-05-21 19:53:19,971	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-05-21 19:53:19,971	[INFO]	(main.py:21)on_ready	robot 「The_PRTS-测试中」 on_ready!
2025-05-21 20:00:43,598	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-05-21 20:00:43,828	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 1775
2025-05-21 20:00:44,521	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-05-21 20:00:44,521	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-05-21 20:00:44,521	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-05-21 20:00:44,521	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-05-21 20:00:44,676	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-05-21 20:00:44,766	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「The_PRTS-测试中」启动成功！
2025-05-21 20:00:44,766	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-05-21 20:00:44,766	[INFO]	(main.py:21)on_ready	robot 「The_PRTS-测试中」 on_ready!
2025-05-21 20:01:11,589	[INFO]	(client.py:162)_bot_login	[botpy] 登录机器人账号中...
2025-05-21 20:01:11,790	[INFO]	(robot.py:65)update_access_token	[botpy] access_token expires_in 1747
2025-05-21 20:01:12,221	[INFO]	(client.py:181)_bot_init	[botpy] 程序启动...
2025-05-21 20:01:12,221	[INFO]	(connection.py:60)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2025-05-21 20:01:12,221	[INFO]	(client.py:242)bot_connect	[botpy] 会话启动中...
2025-05-21 20:01:12,221	[INFO]	(gateway.py:115)ws_connect	[botpy] 启动中...
2025-05-21 20:01:12,367	[INFO]	(gateway.py:142)ws_identify	[botpy] 鉴权中...
2025-05-21 20:01:12,480	[INFO]	(gateway.py:85)on_message	[botpy] 机器人「The_PRTS-测试中」启动成功！
2025-05-21 20:01:12,480	[INFO]	(gateway.py:223)_send_heart	[botpy] 心跳维持启动...
2025-05-21 20:01:12,480	[INFO]	(main.py:21)on_ready	robot 「The_PRTS-测试中」 on_ready!
2025-05-21 20:02:31,809	[INFO]	(main.py:29)on_group_at_message_create	{'id': 'ROBOT1.0_gh8fpwJNjAbqx1DPfhhlYrolBHMZKTiu4HtNcdXy3lVeo8x-X-nDnDJO6.iKLszL6D.E0PpOe2tgrlgOWVr1MQ!!', 'timestamp': '2025-05-21T20:02:32+08:00'}
2025-05-21 20:11:47,701	[ERROR]	(http.py:76)_handle_response	[botpy] 接口请求异常，请求连接: https://sandbox.api.sgroup.qq.com/v2/groups/2078EB9E50A2A3A372D2F03047B60102/messages, 错误代码: 400, 返回内容: {'message': '消息被去重，请检查请求msgseq', 'code': 40054005, 'err_code': 40054005, 'trace_id': '2b6dcbf76150ffd5dae4b91b6d18fe49'}, trace_id:2b6dcbf76150ffd5dae4b91b6d18fe49
2025-05-21 20:15:00,994	[INFO]	(main.py:29)on_group_at_message_create	{'id': 'ROBOT1.0_gh8fpwJNjAbqx1DPfhhlYlovW1ZDtFGjiIAQffsNSaihaCYMqergWtjyOZlU3lSDxdr-veqsM-o0BTCv7Q9sFQ!!', 'timestamp': '2025-05-21T20:15:02+08:00'}
